arrow-core/src/main/kotlin/arrow/core/extensions/eval/apply/EvalApply.kt

```diff
@@ -632,15 +628,15 @@ fun <A, B> Kind<ForEval, A>.product(arg1: Kind<ForEval, B>): Eval<Tuple2<A, B>>
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "product(arg1)",
-  "arrow.core.product"
+    "Eval.mapN(this, fb) { (ab, c) -> Tuple3(ab.a, ab.b, c) }",
+    "arrow.core.Eval"
```

Comment:
```suggestion
    "Eval.mapN(this, fb) { ab, c -> Tuple3(ab.a, ab.b, c) }",
    "arrow.core.Eval", "arrow.core.Tuple3"
```

Idem for other `product` methods